modules/configuring-machine-pool-disk-volume.adoc

```diff
@@ -3,20 +3,18 @@
 // * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
 
 :_mod-docs-content-type: CONCEPT
-[id="configuring_machine_pool_disk_volume{context}"]
+[id="configuring-machine-pool-disk-volume_{context}"]
 = Configuring machine pool disk volume
 
-Machine pool disk volume size can be configured for additional flexibility. The default disk size is 300 GiB. For cluster version 4.13 or earlier, the disk size can be configured to a minimum of 128 GiB to a maximum of 1 TiB. For cluster version 4.14 and later, the disk size can be configured to a minimum of 128 GiB to a maximum of 16 TiB.
+Machine pool disk volume size can be configured for additional flexibility. The default disk size is 300 GiB.
+
+For {rosa-classic-first} clusters version 4.13 or earlier, the disk size can be configured to a minimum of 128 GiB to a maximum of 1 TiB. For cluster version 4.14 and later, the disk size can be configured to a minimum of 128 GiB to a maximum of 16 TiB.
+
+For {hcp-title-first} clusters, the disk size can be configured from a minimum of 75GiB to a maximum of 16,384 GiB.
```

Comment:
The disk size minimum of ROSA 1.2.45  is `128GiB`. And it will update to `75GiB` when ROSA 1.2.46, which hasn't been released. The codes changes only in master branch.

Reply:
Yes.